packages/components/e2e/component-objects/src/main/java/org/talend/component/AppHeaderBar.java

```diff
@@ -152,4 +152,13 @@ public WebElement getHelp() {
     public void clickOnHelp() {
         this.getHelp().click();
     }
+
+    /**
+     * Retrieve an item by its id
+     * @param id
+     * @return the header bar element with the given id
+     */
+    public WebElement getItemById(String id) {
```

Comment:
Hmm not sure that this is the kind of thing we want. We create and maintain generic selectors so they follow any structure change of the component.
Ids should be used to get the component (ex: if you have 2 lists in the page, use the id to identify the list) but to work within the component, you should use the utility function we provide, or else you don't need the component-objects

Reply:
well I did it because of the translation ... So i didn't want to depend on labels as they can change, didn't find better than that.